src/expr.rs

```diff
@@ -1716,9 +1716,11 @@ pub(crate) fn rewrite_field(
         let overhead = name.len() + separator.len();
         let expr_shape = shape.offset_left(overhead)?;
         let expr = field.expr.rewrite(context, expr_shape);
-
+        let is_lit = matches!((*field.expr).kind, ast::ExprKind::Lit(_));
```

Comment:
A minor nit, but could you rewrite this as:
```suggestion
        let is_lit = matches!(field.expr.kind, ast::ExprKind::Lit(_));
```